lnt/testing/profile/profile.py

```diff
@@ -21,18 +21,20 @@ def __init__(self, impl):
         self.impl = impl
 
     @staticmethod
-    def fromFile(f):
+    def fromFile(f, objdump=None):
         """
         Load a profile from a file.
         """
+        if objdump is None:
+            objdump = os.getenv("CMAKE_OBJDUMP", "objdump")
```

Comment:
I just stumbled upon this change, sorry for being late to the party. This seems like a weird place to be special casing on CMake though -- this code is pretty generic and untied to CMake.

Instead, shouldn't we be threading `objdump` through from the various callsites of `fromFile` (there's only a few)?

Reply:
> Instead, shouldn't we be threading objdump through from the various callsites of fromFile (there's only a few)?

Yeah that's what this PR is doing basically. The os.getenv here is just to preserve the existing behaviour from the lines below, which I think affects `lnt profile import`. 

Agrees we can can probably pull the os.getenv call out